docs/changelog.rst

```diff
@@ -11,8 +11,38 @@ The full release history can be viewed `at the GitHub perses releases page <http
 0.10.0 - Release
 ----------------
 
+New command line interface (CLI), many enhancements for the API (specially the ``PointMutationExecutor``) and improved testing.
+
+Bugfixes
+^^^^^^^^
+- Bug in geometry engine when could not generate a successful topology -- fixed by not trying to access it when its ``None``. (`#855 <https://github.com/choderalab/perses/pull/855>`_)
+- Bug generated by changes upstream in the ``openmm`` package -- fixed by adding more padding to the simulation box, it is now 1.1 nm. Openmm issue `#3502 <https://github.com/openmm/openmm/issues/3502>`_. Perses issue `#949 <https://github.com/choderalab/perses/issues/949>`_ (`#953 <https://github.com/choderalab/perses/pull/953>`_)
+- Fixed energy bookkeeping in test of ``HybridTopologyFactory`` when a ring amino acid is involved in transformation. (`#969 <https://github.com/choderalab/perses/pull/969>`_)
+- Avoid changing the global context cache behavior on module imports. Issue `#968 <https://github.com/choderalab/perses/issues/968>`_ (`#977 <https://github.com/choderalab/perses/pull/977>`_).
+- Benchmark free energy plots now shift data to experimental mean. (`#981 <https://github.com/choderalab/perses/pull/981>`_)
+- Recovered logging capabilities respecting the ``LOGLEVEL`` environment variable. Issue `#1018 <https://github.com/choderalab/perses/issues/1018>`_ (`#1032 <https://github.com/choderalab/perses/pull/1032>`_).
+
+
 Enhancements
-------------
+^^^^^^^^^^^^
+- Improved continuous integration (CI) performance. (`#961 <https://github.com/choderalab/perses/pull/961>`_)
+- ``PointMutationExecutor`` now accepts both solute and solvated PDBs. (`#967 <https://github.com/choderalab/perses/pull/967>`_)
+- Tests and examples are now using ``openff-2.0.0`` force field instead of ``openff-1.0.0``. (`#971 <https://github.com/choderalab/perses/pull/971>`_)
+- Use names for fetching the force components of a system, avoiding issues with indices reordering upstream in ``openmm``. Issue `#993 <https://github.com/choderalab/perses/issues/993>`_ (`#976 <https://github.com/choderalab/perses/pull/976>`_ and `#1007 <https://github.com/choderalab/perses/pull/1007>`_)
```

Comment:
```suggestion
- Use names (instead of indices) for fetching the force components of a system, avoiding issues with force reordering upstream in ``openmm``. Issue `#993 <https://github.com/choderalab/perses/issues/993>`_ (`#976 <https://github.com/choderalab/perses/pull/976>`_ and `#1007 <https://github.com/choderalab/perses/pull/1007>`_)
```